Update bigquery extractor docs (#278)
keboola/connection-docs: components/extractors/database/bigquery/index.md

{: .image-popup}
![Screenshot - Name Query](/components/extractors/database/bigquery/bigquery-6.png)

> How to modify your configuration you can read in the [SQL databases section](/components/extractors/database/sqldb/#modify-configuration).

Reviewer: ```suggestion
To learn how to modify your configuration, go to the [SQL databases section](/components/extractors/database/sqldb/#modify-configuration).
```
